Assets/MRTK/SDK/Features/UX/Prefabs/Slate/SlateUGUI.prefab

```diff
@@ -3297,8 +3297,8 @@ MonoBehaviour:
       m_Calls: []
   m_text: UGUI Input Field
   m_isRightToLeft: 0
-  m_fontAsset: {fileID: 11400000, guid: 6a84f857bec7e7345843ae29404c57ce, type: 2}
-  m_sharedMaterial: {fileID: 21202819797275496, guid: 6a84f857bec7e7345843ae29404c57ce,
+  m_fontAsset: {fileID: 11400000, guid: ad601d838b0fb2744937cd5dad890e81, type: 2}
```

Comment:
Is this font change intentional too?

Reply:
Yes, updated to use the same font as the content on the left column. (subheader title 'UGUI Input Field)